File: JavaScript/CWE-798.js
// Hard-coded credentials in JavaScript
const username = "admin";
const password = "password123";

function authenticate(user, pass) {
    if (user === username && pass === password) {
        console.log("Authentication successful!");
    } else {
        console.log("Authentication failed.");
    }
}

// Example usage
authenticate("admin", "password123");

File: GO/CWE-798.go
package main

import "fmt"

// Hard-coded credentials in Go
const (
	username = "admin"
	password = "password123"
)

func authenticate(user, pass string) {
	if user == username && pass == password {
		fmt.Println("Authentication successful!")
	} else {
		fmt.Println("Authentication failed.")
	}
}

func main() {
	// Example usage
	authenticate("admin", "password123")
}
